frontend/rpm/template_test.go

```diff
@@ -209,18 +209,116 @@ func TestTemplateSources(t *testing.T) {
 }
 
 func TestTemplate_Artifacts(t *testing.T) {
-
-	w := &specWrapper{Spec: &dalec.Spec{
-		Artifacts: dalec.Artifacts{
-			SystemdUnits: map[string]dalec.SystemdUnitConfig{
-				"test.service": {},
+	t.Run("test systemd unit postun", func(t *testing.T) {
+		t.Parallel()
+		w := &specWrapper{Spec: &dalec.Spec{
+			Artifacts: dalec.Artifacts{
+				SystemdUnits: map[string]dalec.SystemdUnitConfig{
+					"test.service": {},
+				},
 			},
-		},
-	}}
+		}}
 
-	got := w.PostUn().String()
-	want := `%postun
+		got := w.PostUn().String()
+		want := `%postun
 %systemd_postun test.service
 `
-	assert.Equal(t, want, got)
+		assert.Equal(t, want, got)
+	})
+
+	t.Run("test doc templaing using artifact config", func(t *testing.T) {
```

Comment:
nit for these tests: s/templaing/templating

Reply:
doh...sorry!